charts/cardano-node/values.yaml

```diff
@@ -17,3 +18,6 @@ tolerations:
   operator: Equal
   value: arm64
   effect: NoSchedule
+
+# Enable the PodMonitor for Prometheus Operator
+enablePodMonitor: false
```

Comment:
Something like the following is cleaner. It also more easily allows making other PodMonitor params configurable.

```
podMonitor:
  enabled: false
```